plugins/destination/postgresql/client/spec/spec.go

```diff
@@ -39,6 +39,9 @@ type Spec struct {
 
 	// Option to create specific indexes to improve deletion performance
 	CreatePerformanceIndexes bool `json:"create_performance_indexes,omitempty" jsonschema:"default=false"`
+
+	// Number of times to retry a transaction if a deadlock is detected by Postgres.
+	RetryOnDeadlock int64 `json:"retry_on_deadlock,omitempty" jsonschema:"default=0"`
```

Comment:
We should also add this to the docs (under the docs directory)

Reply:
Docs added in 2b03787.